src/calculateRentalCost.js

```diff
@@ -29,6 +29,24 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let finalPrice = 0;
+  const fullPrice = days * 40;
+
+  if (days < 3) {
+    return fullPrice;
+  }
+
+  if (days < 7) {
+    finalPrice = fullPrice - 20;
+
+    return finalPrice;
```

Comment:
You can return it immediately
```suggestion
    return fullPrice - 20;
```